### PR TITLE
779 biorender demo

### PR DESCRIPTION
**Pull request by matthewcchan**
Corresponds to #779.
Added biorender info and how-to under dasl_demos

### CODE REVIEW

File: _dasldemos/biorender.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+main_authors: matthewcchan
+primary_reviewers: 
+---
+
+[BioRender](https://www.biorender.com/) is a web application that creates scientific figures, illustrations, and diagrams. 
+An institutional license for BioRender Premium is available for Fred Hutch employees and includes unlimited illustrations, thousands of individual icons, graphical assets, templates, and permissions for journal publications.
+
+Below are some examples made by Fred Hutch researchers:
+
+![biorender_transport_example](./assets/biorender_example1.png)
```
Comment (tefirman) [inline _dasldemos/biorender.md:12]:
```suggestion
![biorender_transport_example](/dasldemos/assets/biorender_example1.png)
```

File: _dasldemos/biorender.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+![](https://www.fredhutch.org/en/news/spotlight/2023/11/crd-liang-bloodadv/_jcr_content/root/responsivegrid/image.coreimg.png/1700265944560/crd-liang-bloodadv-image-sized.png)
+Long-term (over 6 years) response evaluation workflow of CD19 CAR T-cell therapy to treat relapsed/refractory (R/R) chronic lymphocytic leukemia (CLL) in a phase I/II clinical trial. [Science Spotlight link.](https://www.fredhutch.org/en/news/spotlight/2023/11/crd-liang-bloodadv.html) Source: Dr. Emily Liang
+
+
+## Getting Started
+To access BioRender, Fred Hutch employees may sign in using their Fred Hutch institutional email or access BioRender via the Apps menu in [Microsoft365](https://www.microsoft365.com/apps?auth=2&home=1).
+
+Login to [BioRender using SSO](https://app.biorender.com/user/sso) and Fred Hutch credentials. 
+![](./assets/biorender_login.png)
```
Comment (tefirman) [inline _dasldemos/biorender.md:28]:
```suggestion
![](/dasldemos/assets/biorender_login.png)
```

File: _dasldemos/biorender.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+Alternatively, users may access BioRender in [Microsoft365](https://www.microsoft365.com/apps?auth=2&home=1).
+![](./assets/biorender_apps.png)
```
Comment (tefirman) [inline _dasldemos/biorender.md:31]:
```suggestion
![](/dasldemos/assets/biorender_apps.png)
```

File: _dasldemos/biorender.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+
+
+## Making Illustrations in the Web Interface
+
+The Gallery the starting point for making and accessing files (illustrations, posters, graphs, etc). As an example, we will make a new illustration file by clicking on `+ Create new`,  then `Illustration`.
+![](./assets/biorender_gallery.png)
```
Comment (tefirman) [inline _dasldemos/biorender.md:37]:
```suggestion
![](/dasldemos/assets/biorender_gallery.png)
```

File: _dasldemos/biorender.md [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+
+
+The left hand panel contains graphical assets that can used for your illustrations. There are over 5000 icons to use. Simply drag and drop an icon to the artboard to use in your illustration.
+![](./assets/biorender_artboard.png)
```
Comment (tefirman) [inline _dasldemos/biorender.md:41]:
```suggestion
![](/dasldemos/assets/biorender_artboard.png)
```

File: _dasldemos/biorender.md [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+
+By clicking on the individual icon, the left hand panel contains several options to adjust color, style, and other graphical aspects. 
+![](./assets/biorender_style.png)
```
Comment (tefirman) [inline _dasldemos/biorender.md:44]:
```suggestion
![](/dasldemos/assets/biorender_style.png)
```

File: _dasldemos/biorender.md [hunk 7]
```diff
@@ -0,0 +1,54 @@
...
+
+
+To export illustrations, click `File` > `Export`. Here, you may set export dimensions, file type, transparency, and resolution. 
+![](./assets/biorender_export.png)
```
Comment (tefirman) [inline _dasldemos/biorender.md:48]:
```suggestion
![](/dasldemos/assets/biorender_export.png)
```

File: _dasldemos/biorender.md [hunk 8]
```diff
@@ -0,0 +1,54 @@
...
+Login to [BioRender using SSO](https://app.biorender.com/user/sso) and Fred Hutch credentials. 
+![](./assets/biorender_login.png)
+
+Alternatively, users may access BioRender in [Microsoft365](https://www.microsoft365.com/apps?auth=2&home=1).
+![](./assets/biorender_apps.png)
+
+
+## Making Illustrations in the Web Interface
+
+The Gallery the starting point for making and accessing files (illustrations, posters, graphs, etc). As an example, we will make a new illustration file by clicking on `+ Create new`,  then `Illustration`.
```
Comment (tefirman) [inline _dasldemos/biorender.md:36]:
```suggestion
The Gallery is the starting point for making and accessing files (illustrations, posters, graphs, etc). As an example, we will make a new illustration file by clicking on `+ Create new`,  then `Illustration`.
```

**Review by tefirman — CHANGES_REQUESTED** [review]
This is great, Matthew, thanks for contributing your BioRender knowledge! The content looks awesome, but I tried building the site locally and noticed that the images weren't quite compiling right with the relative path. I added some suggested changes to switch them to the absolute path (which compiles correctly) and put in one other small grammatical change. Thanks again for submitting this!

**Comment by atombaby** [comment]
The Biorender TOS includes a number of restrictions that would be good to note.  Of particular is 1.e.i.E:

> E) publish Completed Graphics that include BioRender Content in **any publication if you are not cited as the first or co-first author** of the publication due to copyright ownership of the publication being attributed to the first or co-first author. Any exceptions to this restriction must be approved in writing by BioRender.

(emphasis mine).

**Comment by matthewcchan** [comment]
Thank you @tefirman and @atombaby for your comments. I have incorporated your edits in [7cfcdb8](https://github.com/FredHutch/wiki/pull/1010/commits/7cfcdb815790d8ded44688d0dd206edb3cc0751f).

**Review by atombaby — APPROVED** [review]

**Pull request merged**